routes/user/userSecure.js

```diff
@@ -90,4 +92,135 @@ router.put('/permission', async (req, res) => {
     }
 });
 
+// Create new user
+router.post('/', async (req, res) => {
+    try {
+        if (req.body && req.body.firstName && req.body.lastName && req.body.roleID && req.body.email && req.body.password) {
+            if (req.user.roleID == 14139 || req.user.roleID == 21149) {
+                const { firstName, lastName, roleID, email, password } = req.body;
+
+                if (!validateName(firstName)) {

```

Comment:
a lot of these error codes are wrong a 400 means bad request/bad data and a 403 means forbidden/incorrect permissions